tests/data/bats.json

```diff
@@ -202,7 +202,7 @@
                     "group": "Positioner",
                     "sensor_divisor": 1,
                     "sensor_multiplier": 1,
-                    "sensor_current": 17,
```

Comment:
Here, the value looks very different, it must be checked if this is correct or not

Reply:
344.4 is incorrect, looking at the data (it should be 17).